{
  "comments": [
    {
      "key": {
        "uuid": "facab960_9beb8712",
        "filename": "adapter/itest-hibernate-adapter/src/main/java/org/openstack/atlas/util/itest/hibernate/HuApp.java",
        "patchSetId": 20
      },
      "lineNbr": 86,
      "author": {
        "id": 1001830
      },
      "writtenOn": "2015-06-16T19:05:10Z",
      "side": 1,
      "message": "So there\u0027s no rollback anymore?",
      "revId": "3935f65458ff84cbb7c873bb5106daa8394dd117",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "facab960_1b74d766",
        "filename": "adapter/itest-hibernate-adapter/src/main/java/org/openstack/atlas/util/itest/hibernate/HuApp.java",
        "patchSetId": 20
      },
      "lineNbr": 86,
      "author": {
        "id": 1001794
      },
      "writtenOn": "2015-06-16T19:13:01Z",
      "side": 1,
      "message": "This is not code used in production This was for my Jython tests. I removed the auto roll back because higher level code is supposed to abort the transaction.\n\n    But again this code or any code in the itest section is not apart of any of our production code. Notice that originally transactions where rolled back with out warning leading to a bunch of WTFs when attempting to save an object. Now you get an error and its up to you to rollback the session from the Jython command line.",
      "parentUuid": "facab960_9beb8712",
      "revId": "3935f65458ff84cbb7c873bb5106daa8394dd117",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "facab960_bb33eba9",
        "filename": "adapter/itest-hibernate-adapter/src/main/java/org/openstack/atlas/util/itest/hibernate/HuApp.java",
        "patchSetId": 20
      },
      "lineNbr": 86,
      "author": {
        "id": 1001830
      },
      "writtenOn": "2015-06-16T19:18:40Z",
      "side": 1,
      "message": "Ah, whoops, missed that. I tried to ignore pretty much any of the itest/jython files, sorry.",
      "parentUuid": "facab960_1b74d766",
      "revId": "3935f65458ff84cbb7c873bb5106daa8394dd117",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "facab960_9b39c7a6",
        "filename": "logs/logs-hdfs-cli/src/main/java/org/openstack/atlas/logs/itest/HdfsCli.java",
        "patchSetId": 20
      },
      "lineNbr": 852,
      "author": {
        "id": 1001830
      },
      "writtenOn": "2015-06-16T19:05:10Z",
      "side": 1,
      "message": "In the help printout above, you call this \"cdjar\", not plural. I think you\u0027re the only one that uses these CLIs, but it might be confusing.",
      "revId": "3935f65458ff84cbb7c873bb5106daa8394dd117",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "facab960_fb12d307",
        "filename": "logs/logs-hdfs-cli/src/main/java/org/openstack/atlas/logs/itest/HdfsCli.java",
        "patchSetId": 20
      },
      "lineNbr": 852,
      "author": {
        "id": 1001794
      },
      "writtenOn": "2015-06-16T19:13:01Z",
      "side": 1,
      "message": "fixing the misspelling.",
      "parentUuid": "facab960_9b39c7a6",
      "revId": "3935f65458ff84cbb7c873bb5106daa8394dd117",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    }
  ]
}